alpine/Dockerfile

```diff
@@ -38,7 +38,7 @@ RUN set -eux; \
     && mv /kong/usr/local/* /usr/local \
     && mv /kong/etc/* /etc \
     && rm -rf /kong \
-    && apk add --no-cache libstdc++ libgcc openssl pcre perl tzdata libcap zip bash zlib zlib-dev git ca-certificates \
+    && apk add --no-cache perl libgcc zlib-dev bash unzip outils-md5 \
```

Comment:
Any thoughts on removing bash, unzip, and outils-md5? These seem unnecessary. Same perhaps for libgcc?

Reply:
unzip and outils-md5 are needed to add plugins, bash is needed for the smoke tests. We could remove those packages by changing, removing or reducing scope of the smoke tests and documenting that they are needed for adding plugins.

Reply:
`ca-certificates` has become a somewhat critical package because of let's encrypt. We've had a variety of build break because they either didn't have this at all or had an older version. LE adjusted their root certificates and suddenly the image can't securely talk to a large amount of the internet

Just providing information no thoughts on if it should stay or not

Reply:
cross-linking past conversation regarding `zlib-dev` https://github.com/docker-library/official-images/pull/8337#issuecomment-656859264